services/ux-backend/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	handler "github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	"k8s.io/klog/v2"
+)
+
+func main() {
+
+	klog.Info("Starting ux backend server")
+
+	var err error
+	var listenPort int
+	var tokenLifetimeInHours int
+
+	defaultTokenLifetimeInHours := 48
+	tokenLifetimeInHoursAsString := os.Getenv("ONBOARDING_TOKEN_LIFETIME")
+	if tokenLifetimeInHoursAsString == "" {
+		klog.Infof("No user-defined token lifetime provided, defaulting to %d ", defaultTokenLifetimeInHours)
+		tokenLifetimeInHours = defaultTokenLifetimeInHours
+	} else if tokenLifetimeInHours, err = strconv.Atoi(tokenLifetimeInHoursAsString); err != nil {
+		klog.Errorf("Malformed user-defined Token lifetime: %s. shutting down", tokenLifetimeInHoursAsString)
+		os.Exit(-1)
+	}
+
+	klog.Infof("generated token will be valid for %d hours", tokenLifetimeInHours)
+
+	defaultListeningPort := 8080
+	listenPortAsString := os.Getenv("UX_BACKEND_PORT")
+	if listenPortAsString == "" {
+		klog.Infof("No user-defined server listening port provided, defaulting to %d ", defaultListeningPort)
+		listenPort = defaultListeningPort
+	} else if listenPort, err = strconv.Atoi(listenPortAsString); err != nil {
+		klog.Errorf("Malformed user-defined listening port: %s. shutting down", listenPortAsString)
+		os.Exit(-1)
+	}
```

Comment:
Having all of this code in the main func context is cumbersome and not scalable.
Can we move this logic into a separate func:
``` func loadAndValidateServerConfig() *serverConfig {}```

Then `serverConfig` can be defined as:
```
type serverConfig struct {
    listenPort int
    tokenLifetimeInHours int
} 
```

This will allow us to extend the config and isolate the approach to the config (maybe in the future we will decide to have all of the config in a config map and not env vars)